portfolio/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,208 @@
+/* Contains the styling that applies to all pages" */
+
+
+* {
+    font-family: Roboto;
+    text-decoration: None;
+}
+
+/*Page level container*/
+body {
+  background-color:white;
+}
+
+.meta-header {
+    display: flex;
+}
+
+.upper-area {
+    flex-grow: 8;
+}
+
+.header {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    justify-content: space-between;
+    align-items: center;
+
+    border-bottom-style: solid;
+    border-bottom-color: black;
+    border-bottom-width: 2px;
+}
+
+.title-section > * {
+    margin-right: 20px;
+}
+
+.logo {
+    height: 100px;
+    width: 100px;
+    border-radius: 50%;
+    padding-right: 10px;
+    padding-top: 10px;
+}
+
+.title {
+    font-weight: bold;
+    font-size: 40px;
+    color: black;
+}
+
+.nav-section > * {
+    margin-right: 40px;
+}
+
+.nav-links {
+    font-size: 20px;
+    color: black;
+}
+
+.title:hover, .nav-links:hover {
+    color: white;
+    background-color: black;
+}
+
+.nav-section-mobile {
+    display: none;
+    position: relative;
+    margin-right: 40px;
+    align-items: center;
+}
+
+.dropdown-links {
+    display: none;
+    position: absolute;
+    background-color: #f9f9f9;
+    min-width: 160px;
+    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+    padding: 12px 16px;
+    z-index: 1;
+}
+
+.nav-section-mobile:hover .dropdown-links {
+    display: block;
+}
+
+@media (max-width: 700px) {
+    .nav-section {
+        display: none !important;
+    }
+    .nav-section-mobile {
+        display: inline-block;
+    }
+}
+
+.footer {
+    display: flex;
+    justify-content: center;
+}
+
+.fa {
+    margin: 10px;
+    padding: 20px;
+    font-size: 30px;
+    border-radius: 50%;
+}
+
+.fa-google {
+  background: #dd4b39;
+  color: white;
+}
+
+.fa-linkedin {
+  background: #007bb5;
+  color: white;
+}
+
+.icon:hover {
+    opacity: 0.7;
+}
+
+/* Section level container
```

Comment:
nit: It is bad practice to leave commented out code. If you will need it later you can always find it in history.

Reply:
I agree. This was removed in a later commit.